### PR TITLE
add phpmyadmin to docker to make accces to database easy

### PR DESCRIPTION
**Pull request by walidjlidi**
i really appreciate your effort in this repo and i wanna to add a small feature

### CODE REVIEW

**Comment by Maykonn** [comment]
Done. Thanks!

**Comment by Maykonn** [comment]
Please, assist this project given a star and following haha. :)

**Comment by walidjlidi** [comment]
thank you for accepting my PR :)

File: docker/docker-compose.example.yml [hunk 1]
```diff
@@ -45,4 +45,12 @@ services:
...
     volumes: 
       - "../app:/var/www/app:cached"
+phpmyadmin:
+    image: phpmyadmin/phpmyadmin
+    ports:
+        - "99:80"
+    networks: 
+      sf_network: ~
+    links:
+        - db
```
Comment (galion96) [inline docker/docker-compose.example.yml:55]: Is there a missing space here?

Comment (galion96) [inline docker/docker-compose.example.yml:55]: Is this inlined properly compared to the file above?

Comment (walidjlidi) [reply docker/docker-compose.example.yml:55]: yes you are right :)

**Pull request merged**